curve25519-dalek/src/backend/mod.rs

```diff
@@ -128,6 +128,30 @@ impl VartimePrecomputedStraus {
         }
     }
 
+    pub fn len(&self) -> usize {
```

Comment:
Only note: could we have docs for these methods?

Reply:
Done!